hw/top_earlgrey/doc/_index.md

```diff
@@ -16,24 +16,33 @@ This is not a specification of the final implementation.
   - DM (debug module)
   - PLIC (platform level interrupt controller)
   - U/M execution modes (user/machine)
+  - Enhanced Physical Memory Protection (ePMP)
 - Memory contents
-  - 512kB emulated eFlash for code and data storage
-  - 64kB SRAM for data storage
-  - 16kB ROM for secure boot code storage
+  - 1024kB eFlash
+  - 128kB SRAM
+  - 4KB Always ON (AON) retention SRAM
+  - 32kB ROM
 - Security peripherals
   - Flash controller
-  - AES module with support for ECB, CBC, CFB, OFB and CTR modes of operation
-  - SHA-256/HMAC module
-  - Basic alert responder
-  - (coming soon) emulated TRNG entropy source
+  - AES
+  - SHA-256/HMAC
+  - KMAC
```

Comment:
do you think it's worth saying SHA-3/KMAC just like above?

Reply:
Done.